CHANGELOG-ABP.md

```diff
@@ -0,0 +1,30 @@
+# Release Notes
+## Chromium 68.0
+* Updated to Chromium `68.0.3440.70`
+* [Use Chromium Android NDK](https://gitlab.com/eyeo/adblockplus/chromium/issues/26) when building, instead of downloading another copy
+* [Add support for](https://issues.adblockplus.org/ticket/6205) `libadblockplus-android` as a Gradle dependency
```

Comment:
We need to remove that line: we don't use it in Chromium. (still extracting the sources and compiling them while building Chromium).